__tests__/nc-news.test.js

```diff
@@ -149,8 +149,52 @@ describe("200 - GET /api/articles/article_id/comments", () => {
     return request(app)
       .get("/api/articles/1/comments")
       .then((response) => {
-        const articles = response.body.articleComments;
-        expect(articles).toBeSortedBy("created_at", { descending: true });
+        const articleComments = response.body.articleComments;
+        expect(articleComments).toBeSortedBy("created_at", {
+          descending: true,
+        });
+      });
+  });
+});
+
+describe("201 - POST - /api/articles/:article_id/comments", () => {
+  it("should respond with a 201 response and add a comment to an article with the following properties: username, body. And should respond with the posted comment", () => {
+    const newCommentObj = {
+      username: "rogersop",
+      body: "do you like my body?",
+    };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(newCommentObj)
+      .expect(201)
+      .then((response) => {
+        const postedComment = response.body.newComment;
+        postedComment.forEach((comment) => {
```

Comment:
There should just be a single comment returned so no need to do a forEach. In your model if you return rows[0] you will just return the comment, not the array